FreeRTOS/Demo/RISC-V_RV32_SiFive_HiFive1-RevB_IAR/main.c

```diff
@@ -242,7 +242,6 @@ void vApplicationMallocFailedHook( void )
      * to query the size of free heap space that remains (although it does not
      * provide information on how the remaining heap might be fragmented). */
     taskDISABLE_INTERRUPTS();
-    __asm volatile( "ebreak" );

```

Comment:
Similarly here as well. Would it not be better to inform the user of the malloc failure as soon as it happens by stopping the debugger? By removing this, the user will keep waiting for something to happen and eventually hit 'pause' on the debugger to find that the code was stuck on this loop the whole time!